content/en/getting_started/site/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Datadog Sites
+kind: documentation
+---
+
+Datadog offers different sites world-wide. Each site is completely independent, and you cannot share data across sites. Each site gives you benefits, for example, government security regulations, or allows you to store your data in specific locations around the world.
+
+| Site    | Site URL                   | Location |
+|---------|----------------------------|----------|
+| US1     | https://app.datadoghq.com/ | US       |
+| US3     | https://us3.datadoghq.com  | US       |
+| EU1     | https://app.datadoghq.eu/  | EU       |
+| US1-FED | https://app.ddog-gov.com   | US       |
+
+## How do I tell which Datadog site I am on?
+
+You can identify which site you are on by matching your Datadog application URL to the site URL in the table above.
+
+{{< img src="getting_started/site/site.png" alt="Image of the site URL in your browser"  >}}
+
+Since the Datadog sites may support different Datadog functionality depending on that instance's security requirements, the documentation website allows you to select which site you need information for by using the dropdown site selector that is on the right side of all of the documentation pages.
+
+{{< img src="getting_started/site/site-selector.png" alt="Image of the site selector on a documentation page"  >}}
+
+## The Datadog for Government site
+
+The Datadog for Government site (US1-FED) is meant to allow US government agencies and partners to monitor their applications and infrastructure. For information about the Datadog for Government site security and compliance controls and frameworks, and how it supports FedRAMP, see our [security page][1].
```

Comment:
```suggestion
The Datadog for Government site (US1-FED) is meant to allow US government agencies and partners to monitor their applications and infrastructure. For information about the Datadog for Government site's security and compliance controls and frameworks, as well as how it supports FedRAMP, see the [security page][1].
```

man just trying too make that sentence easier to parse is rough... i wanna like, diagram it